js/data_v4.js

```diff
@@ -5,6 +5,8 @@ var THIRTY_DAYS_IN_SECONDS = 2592000;
 var PAINT_TIME_THRESHOLD = 300000;
 var CONFIG_URL = 'js/config.json?';
 var prefs = null;
+// The following is the list of ping types we will display in the page.
+var SUPPORTED_PING_TYPES = ["main", "crash-ping"];
```

Comment:
`ACCUMULATED_PING_TYPES` or so?
Make it a member of `MainPingAccumulator` for more clarity?